soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -126,6 +147,61 @@ constexpr std::array<HintSetting, 4> hintSettingTable{{
   },
 }};
 
+Text AutoFormatHintText(Text unformattedHintText) {
+  std::array<std::string, LANGUAGE_MAX> strings;
+  for (int i = 0; i < LANGUAGE_MAX; i++) {
+    std::string textStr = unformattedHintText.GetForLanguage(i);
+    // RANDOTODO: don't just make manual exceptions
+    bool needsAutomaicNewlines = true;
```

Comment:
This actually isn't my typo. Copied it in from spoiler_log.cpp, since I actually needed the formatting it applied earlier than when it entered the spoiler file after changes.

Also I think this only appeared as a change in this PR because it was opened before the rando-gap-bridging PR got merged, so it showed all of those changes as well. I've since rebased so I don't think it actually shows up as a change anymore.

Also it's out of scope here but the only reason we actually ever needed that variable is due to a bug. So I'll have a separate cleanup PR in the near future that will include that bugfix which will result in that line no longer existing.